src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -476,5 +496,7 @@ def _handle_command(self, _, cmd):
             return self._create_osd(_, cmd)
         elif cmd['prefix'] == "orchestrator osd remove":
             return self._remove_osd(cmd)
+        elif cmd['prefix'] == "orchestrator host ls":
```

Comment:
https://github.com/noahdesu/ceph/blob/8e108d198fa8f2fe6b1a152659b00b316df3e31f/src/pybind/mgr/orchestrator_cli/module.py#L484-L489

Reply:
@jmolmo i think the easiest thing to do is to cherry pick that commit from my branch. i should have a PR up in a day or two for this, but I'm running behind scheule!

Reply:
@noahdesu no problem... for me the important thing is to have implemented/checked the ansible module functionality... and to put in common things (@sebastian-philipp helped me to do that with his comments:-).
We can even wait for your changes and then rebase this PR.

Reply:
I have used the @noahdesu code.